js/live-stream-embed.js

```diff
@@ -2,84 +2,7 @@
 
 var ipfs_gateway_self = 'https://ipfs-gateway.live.mesh.world'; // IPFS gateway of this node
```

Comment:
do we want to set gateways in the config? maybe premature

Reply:
2019 :)